gbdk-lib/examples/gb/banks_autobank/Makefile

```diff
@@ -1,30 +1,22 @@
-LCC	= ../../../bin/lcc -Wa-l -Wl-m
-BANKPACK  = ../../../bin/bankpack
+LCC	= lcc -Wa-l -Wl-m
```

Comment:
Same as comment above about keeping relative path

Reply:
Also probably good to add the no-ice output (-Wl-j) to harmonize it with other examples

Reply:
Ah whoops - that might have just been me fat fingering it